.github/ISSUE_TEMPLATE/governance-review-request.yaml

```
@@ -1,158 +1,166 @@
-    validations:
-      required: true
-
-  - type: textarea
-    attributes:
-      label: Governance Execution Examples
-      description: |
-        Do you have any examples of execution of the governance? Examples could be:
-        - Voting
-        - Promoting a contributor / demoting a maintainer
-        - Security response
-        - Meeting recordings
-    validations:
-      required: true
-
-  - type: textarea
-    attributes:
-      label: Governance Evolution
-      description: |
-        Governance evolution is the observable changes and improvements the project makes to its governance over the project's lifespan. It is expected that changes will occur over the project's life and that such changes are iterative, tested, and adjusted.
-
-        If there are any major milestones in the project's governance over time, please describe them here with supporting URL links.
-    validations:
-      required: true
+#         If these documents are already linked from an Incubation or Graduation application, you may simply
+#         refer to that document by linking it here, and fill in only the documents not listed there.
+#     validations:
+#       required: true
+
+#   - type: textarea
+#     attributes:
+#       label: Governance Execution Examples
+#       description: |
+#         Do you have any examples of execution of the governance? Examples could be:
+#         - Voting
+#         - Promoting a contributor / demoting a maintainer
+#         - Security response
+#         - Meeting recordings
+#     validations:
+#       required: true
+
+#   - type: textarea
+#     attributes:
+#       label: Governance Evolution
+#       description: |
+#         Governance evolution is the observable changes and improvements the project makes to its governance over the project's lifespan. It is expected that changes will occur over the project's life and that such changes are iterative, tested, and adjusted.
+
+#         If there are any major milestones in the project's governance over time, please describe them here with supporting URL links.
+#     validations:
+#       required: true
+
+#   - type: markdown
+#     attributes:
+#       value: |
+#         ## Additional information
+
+#   - type: textarea
+#     attributes:
+#       label: Any specific aspects of your governance structure are you seeking feedback on?
+
+#   - type: textarea
+#     attributes:
+#       label: Do you have any concerns or specific areas where you feel your governance could be improved?
+
+#   - type: textarea
+#     attributes:
+#       label: Additional notes and resources
+#       description: Please add any additional notes and resources here.
 
+body:
   - type: markdown
     attributes:
       value: |
-        ## Additional information
+        This template has moved to the TOC repo. 
+        Please complete the Governance Review Request [here](https://github.com/cncf/toc/issues)
```

```suggestion
        Please complete the [Governance Review Request](https://github.com/cncf/toc/issues) in the TOC repository.
```
Sorry - My personal pet peeve is linking the word "here" 🤣

---

Aside from the correction, this is the only part of the text edit we need.  None of the rest of the commenting.